includes/Lister/Lister.php

```diff
@@ -1301,4 +1318,24 @@ protected function joinSectionTagPieces( $piece, $start, $end ) {
 	public function getRowCount() {
 		return $this->rowCount;
 	}
+
+	
+	public function getPageImage( int $pageID ) {
+
+		$dbl = MediaWikiServices::getInstance()->getDBLoadBalancer();
+		$dbr = $dbl->getConnection( DB_REPLICA );
+		//in the future, a check could be made for page_image too, but page_image_free is the default, should do for now
+		$propValue = $dbr->selectField( 'page_props', // table to use
+					'pp_value', // Field to select
+					[ 'pp_page' => $pageID, 'pp_propname' => "page_image_free" ], // where conditions 
+					__METHOD__
+		);
+		if ( $propValue === false ) {
```

Comment:
We are returning $propValue below this, here we are returning $propValue here if it is false. This is unnecessary since it will return the same value below ($propValue, including false if that is what it is, so this whole if statement is unnecessary.

Reply:
Didn't catch this before, done now.